packages/nvidia_nat_langchain/src/nat/plugins/langchain/langsmith/langsmith_optimization_callback.py

```
@@ -0,0 +1,483 @@
+                # Retroactively tag the existing latest commit
+                if commit_tags:
+                    try:
+                        response = self._client.request_with_retries("GET",
+                                                                     f"/commits/-/{repo_name}/",
+                                                                     params={
+                                                                         "limit": 1, "offset": 0
+                                                                     })
+                        commits = response.json().get("commits", [])
+                        if commits:
+                            self._client._create_commit_tags(f"-/{repo_name}", commits[0]["id"], commit_tags)
+                    except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)